app/views/provider/admin/user/access_tokens/_form.html.slim

```diff
@@ -7,3 +7,14 @@
 = form.input :permission, as: :patternfly_select,
                           collection: @access_token.available_permissions,
                           include_blank: false
+
+- if @access_token.persisted?
+  .pf-c-form__group
+    .pf-c-form__group-label
+      label.pf-c-form__label
+        span.pf-c-form__label-text
+          = t('.expires_at')
+    .pf-c-form__group-control
+      = @access_token.expires_at.present? ? l(@access_token.expires_at) : t('provider.admin.user.access_tokens.no_expiration')
```

Comment:
I am wondering if we could add a root-level `access_tokens` entry with `expires_in`, `expires_at` and `no_expiration` keys?... 
They seem quite generic (even though we currently don't use them in other contexts), so I don't like putting this whole path :upside_down_face: 

But that's just an idea, not a call to action.

Reply:
Done https://github.com/3scale/porta/pull/3943/commits/4036f22506bd63824385258e798d9d3fad484279